src/session/session_dhandle.c

```diff
@@ -438,7 +438,7 @@ __wt_session_lock_checkpoint(WT_SESSION_IMPL *session, const char *checkpoint)
 	 * files, since changes to the underlying file are visible to the in
 	 * memory pages.
 	 */
-	WT_ERR(__wt_cache_op(session, NULL, WT_SYNC_DISCARD));
+	WT_ERR(__wt_cache_op(session, NULL, WT_SYNC_CLOSE));
```

Comment:
This change didn't get reverted when you brought back `WT_SYNC_DISCARD`, was that deliberate?